src/drivers/mct/cime_config/config_component.xml

```diff
@@ -1933,10 +1933,17 @@
     <desc>ROOTPE (mpi task in MPI_COMM_WORLD) for each component</desc>
   </entry>
 
+  <entry id="COUPLER_COUNT">
+    <type>integer</type>
+    <default_value>1</default_value>
+    <group>mach_pes</group>
+    <file>env_mach_pes.xml</file>
+    <desc>Number of couplers in ensemble: if COUPLER_COUNT > 1 then NINST must be 1 for all components.</desc>
```

Comment:
I think this should say something more than this: My understanding is that this effectively sets the number of total instances of the system. e.g., if COUPLER_COUNT is 5, then we have 5 couplers, 5 atm instances, 5 lnd instances, etc.